### ISSUE TITLE
Convert .ckpt files to frozen model .pb

### ISSUE THREAD

**Issue by abdou31** [imohitch/DeepLearningOnAndroid#1]
Hello, 
I have trained a CNN Model to predict and draw landmarks on the eye region.
I have .ckpt files and I want to produce .pb frozen model.
The problem that I don't know the output nodes model, so I have tried a solution but I didn't know that can work or no, 

+ My solution is that I have a .pb pre-trained model and I have a python script that can print nodes names from .pb file. 

I get a whole number of nodes names.
When I take those nodes names and I set them to produce my trained model, I get an error that > import/.. is not in the graph

So I have taken only logits nodes and the result that I get .pb frozen model with size 28 Mo.
How can I know that I have taken ( nodes names ) are the output nodes names?
